extensions/ArchLinux/modules/skins/timeless.less

```diff
@@ -0,0 +1,96 @@
+@import '../arch_definitions';
+
+/* disable default mediawiki logo  */
+#p-logo {
+    display: none !important;
+}
+
+/* make room for archnavbar and disable fixed searchbar */
+body, #mw-content-container {
+    margin-top: 0;
+    background-color: @body-background-color;
+}
+#mw-content-container {
+    border-bottom: none;
+    padding-top: .5em;
+}
+#mw-header-hack, #mw-header-nav-hack {
+    display: none !important;
+}
+#mw-header-container {
+    position: initial;
+}
+#mw-content {
+    border: @content-border-style;
+}
+#content {
+    border: none;
+    background: transparent;
+}
+
+/* z-order fix */
+#personal-inner {
+    z-index: 200;
+}
+
+/* integrate footer into page background */
+#mw-footer {
+    padding-left: 16em;
+    padding-right: 18em;
+}
+#mw-footer > * {
+    font-size: .8em;
+}
+.mw-footer-container {
+    border-top: none;
+    line-height: 1;
+    color: inherit;
+    box-shadow: none;
+}
+.mw-footer-container a {
+    color:#3366cc;
+}
+.mw-footer-container a:hover {
+    color:#2952a3;
+}
+.mw-footer-container a:visited {
+    color:#2a4b8d;
+}
+.mw-footer-container a:visited:hover {
+    color:#1e3666;
+}
+
+
+/* fixes for specific widths */
+@media screen and (max-width:1339px) {
+    #mw-footer {
+        padding-right: 1em;
+    }
+}
+@media screen and (min-width:851px) and (max-width:1099px) {
+    #mw-footer {
+        padding-left: 1em;
+    }
+    #mw-content {
+        margin-top: 1.75em;
+    }
+}
+@media screen and (max-width:850px) {
+    #mw-footer {
+        padding-left: 1em;
+    }
+    #p-logo-text, #site-navigation h2, #site-tools h2, #user-tools h2 {
+        top: 75px;
+    }
+    .sidebar-inner, .dropdown {
+        top: 7.75em;
+    }
+    #menus-cover {
+        top: 65px;
+    }
+}
+
+/* prefer scrollbars instead of wrapping for code blocks */
+pre {
+    white-space: pre;
+}
```

Comment:
If this is desired, IMHO it should instead be fixed in the templates.

Reply:
All right, I'll remove that commit.

Reply:
We actually already have this in [MediaWiki:Common.css](https://wiki.archlinux.org/title/MediaWiki:Common.css).

Reply:
> We actually already have this in [MediaWiki:Common.css](https://wiki.archlinux.org/title/MediaWiki:Common.css).

And it gets overwritten by arch_common.less in most cases, so that change was even more useless.

https://github.com/archlinux/archwiki/blob/237e9018b4c4a4a80057aade8055b8c609af7b5d/extensions/ArchLinux/modules/arch_common.less#L41-L46

Reply:
It was first added in 20a09e42f3a17c3ec07685611b9652ad0334d2ac and 93ba4fa2336dc5321e96d451f9ee03876aa79928.

:thinking: If you want to change it, I think it should be done in a separate PR.